braindecode/models/eegconformer.py

```diff
@@ -149,15 +154,18 @@ def __init__(
             att_drop=att_drop_prob)
 
         self.classification_head = _ClassificationHead(
```

Comment:
```suggestion
        self.fc = _FullyConnected(
```
The name `_ClassificationHead` does not make any more sense